### PR TITLE
fix: [CDS-74343]: registryId field added in ECR trigger

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots
image

NA

https://github.com/harness/harness-core-ui/assets/96036463/290a1fa8-9c4f-4b4b-9e71-c0b5c3e2bce9

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by PratyushGar — APPROVED** [review]

File: src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/ECRArtifact/ECRArtifact.tsx [hunk 1]
```diff
@@ -96,6 +96,18 @@ export function ECRArtifact({
                   placeholder={getString('select')}
                 />
               </div>
+              <div className={css.imagePathContainer}>
```
[the remark this answers is not in the document]
Comment (satyam-harness) [reply src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/ECRArtifact/ECRArtifact.tsx:99]: Changes are done

**Comment by satyam-harness** [comment]
retrigger titlecheck

**Comment by satyam-harness** [comment]
Coverage ticket -> https://harness.atlassian.net/browse/CDS-78152

**Review by vikrant-harness — APPROVED** [review]

File: src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/ECRArtifact/ECRArtifact.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (kalyansai-harness) [reply src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/ECRArtifact/ECRArtifact.tsx:99]: @pankajsingh-harness are you okay with the changes?

**Review by pankajsingh-harness — APPROVED** [review]

**Review by Eric761 — APPROVED** [review]

**Pull request merged**